Documentation/user-guides/webhook.md

```diff
@@ -37,9 +37,7 @@ flags need to be added to the Prometheus Operator deployment:
 
 ## Deploying the admission webhook
 
-Two variants of the admission webhook are available: a validating webhook and a
-mutating webhook. Both reject invalid `PrometheusRule` resources. The mutating
-variant also adds annotations to validated `PrometheusRule`s
+Two webhook endpoints are available for `PrometheusRule` resources, `/admission-prometheusrules/validate` and `/admission-prometheusrules/mutate`.  The endpoint `/admission-prometheusrules/validate` rejects rules that are not valid prometheus config. The endpoint `/admission-prometheusrules/mutate` ensures that integers and boolean yaml data elements are cooerced into strings (https://github.com/prometheus-operator/prometheus-operator/pull/3230).
 
```

Comment:
I am not sure that we need to include the link back to the implementation in the docs, wdyt?

Reply:
I've removed it.